tests/test_req_vars.py

```diff
@@ -622,6 +622,7 @@ def test_attribute_headers(self):
         auth = 'HMAC_SHA1 c590afa9bb59191ffab30f223791e82d3fd3e3af'
         agent = 'testing/1.0.1'
         default_agent = 'curl/7.24.0 (x86_64-apple-darwin12.0)'
+        referer = 'google.com'
```

Comment:
[nitpick] Actual referer values would be full or partial URIs, e.g.:

```
https://example.com/something/something
```

or

```
/partial/uri
```

Not a big deal, since the property doesn't do any validation, but if you feel like changing it I wouldn't complain. :)